helpers.py

```
@@ -16,7 +16,8 @@ def escape(s):
             ("%", "~p"), ("#", "~h"), ("/", "~s"), ("\"", "''")]:
             s = s.replace(old, new)
         return s
-    return render_template("apology.html", top=escape(top), bottom=escape(bottom))
+
+    raise render_template("apology.html", top=escape(top), bottom=escape(bottom)), 400
```

Why `raise` instead of `return`?

---

Oops 😬 

I previously did this by raising a `BadRequest` exception, it's just an artifact of that. My bad!